src/Analyzer/Utils.h

```diff
@@ -50,6 +50,9 @@ std::optional<bool> tryExtractConstantFromConditionNode(const QueryTreeNodePtr &
   */
 void addTableExpressionOrJoinIntoTablesInSelectQuery(ASTPtr & tables_in_select_query_ast, const QueryTreeNodePtr & table_expression, const IQueryTreeNode::ConvertToASTOptions & convert_to_ast_options);
 
+/// Extract table expressions from tree
+QueryTreeNodes extractTrueTableExpressions(const QueryTreeNodePtr & tree);
```

Comment:
It's not really clear what `True` refers to and what is the essential difference from `extractTableExpressions`, consider clarifying comment